include/zephyr/dt-bindings/adc/b91-adc.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 Telink Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_ADC_B91_ADC_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_ADC_B91_ADC_H_
+
+#include <zephyr/dt-bindings/dt-util.h>
```

Comment:
this file is not using this header, please remove it

Reply:
Done.